kubernetes/samples/scripts/create-weblogic-domain/model-in-image/create-domain-resource.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+# Copyright (c) 2019, 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+#
+# This script deploys the domain resoure in WORKDIR/mii-DOMAIN_UID.yaml.
```

Comment:
Oh, this script deploys the domain resource... I was thinking that this script creates the domain resource yaml based on the model.  I don't really need a script for the customer to `kubectl apply -f domain.yaml`.  Let me see what else is here.

Reply:
The sample doc that walks through the sample will clearly include the exact apply command and won't emphasize this script.  In fact, the sample doc will do pretty much the same for all of the scripts. The script is mainly intended for automation/testing - which are core requirements that aren't directly related to using this branch as a sample.